src/components/chart_04/chart_04.js

```diff
@@ -86,6 +86,8 @@ class Chart04 extends Component {
   }
 
   componentDidUpdate() {
+    this.labels = this.props.values.labels;
+
     //const dataSet1 = compileDataSet(labels, this.props.data, colors);
     //change(dataSet1, false);
     const dataSet1 = this.compileDataSet(
```

Comment:
whats happening exavtly here lline 93

Reply:
That was necessery because labels was first initialized in the constructor, so I had to reassign it in the componentDidUpdate since the constructor will be only called once.